.github/workflows/label-ready-approved.yml

```diff
@@ -0,0 +1,13 @@
+on: pull_request_review
```

Comment:
Apparently pull_request_review won't access the secrets, we have a proposed workaround here:

https://stackoverflow.com/questions/67247752/how-to-use-secret-in-pull-request-review-similar-to-pull-request-target